ng-installation.adoc

```diff
@@ -15,90 +15,114 @@ still available via the usual CLI commands.
+TIP: Backup NG features a built-in scheduling system and does not rely on the system's cron service. At the end of the initialisation process, old backup-related crontab lines will be automatically removed.
 
 [[backup-ng-initialisation]]
 Backup NG Initialization
 ~~~~~~~~~~~~~~~~~~~~~~~~
 Before initializing the Backup NG module, make sure you have enough space on
-the storage you wish to store the backup in. The average size of the backup is
+the storage where you will store the backup. The average size of the backup is
 50-70% of the nominal total quota of all mailboxes.
 
-To initialize the Backup NG module:
+*To initialize the Backup NG module:*
 
 * Access the Zimbra Administration Console.
 * Enter the "Network Modules NG" section on the left menu.
 * Enter the "Backup" section.
-* Set the "Backup Path" to whatever directory you wish to store your backup into.
-* Click on the "Initialize" button
+* Set the "Backup Path" to the directory where you will store your backup.
+* Click the "Initialize" button - this will trigger a SmartScan to initialize the Backup Path
 
 [[mobile-ng]]
 Switching to Mobile NG
 ----------------------
 Switching to the new Mobile NG is a simple process that will activate the new
-mobile handlers, deactivate the old ones and switch the synchronization control
+mobile handlers and deactivate the old ones. This will also switch the synchronization control
 over to Mobile NG from the legacy Zimbra Mobile.
-Until the initialization is completed, the old mobile engine will be active.
+Until the initialization is complete, the old mobile engine will be active.
 
 [[what-happens-after-the-switch]]
-What happens after the switch
+What Happens after the Switch
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 After switching to Mobile NG, all existing syncstates will be invalidated,
-and all connected devices will automatically re-synchronize all of their data
+and all connected devices will automatically re-synchronise all of their data
 through the new engine.
 
-WARNING: Since the switch will force all connected devices to re-synchonize all
+WARNING: *Since the switch will force all connected devices to re-synchonise all
 of their data, make sure to alert your users beforehand to make sure that they have
 wifi coverage or enough traffic on their mobile data plans.
+ +
+ +
+Furthermore, the switch might lead to an abrupt increase in the number of connections to the server, and consequently its load, due to the resynchronisation of all devices.*
 
 The switch is completely transparent to end users, and no user interaction
- will be prompted or required.
+ should be prompted or required, but being the Exchange ActiveSync protocol mostly client-driven different behaviours might be experienced, such as:
+
+* Device not synchronising until user's action (e.g. opening the email client).
+* Device requiring a restart.
+* Device not synchronising until the user re-enter its username and password in the account's settings.
```

Comment:
Suggest change "user re-enter its" to "user re-enters their"

Reply:
@zextras-contrib have these changes suggested by @fatkudu been merged into this PR?